normandy/base/api/views.py

```diff
@@ -79,6 +87,75 @@ def get(self, request):
         )
 
 
+class UserViewSet(viewsets.ModelViewSet):
+    """Viewset for managing users."""
+
+    queryset = User.objects.all()
+    serializer_class = UserWithGroupsSerializer
+    permission_classes = (CanChangeUser,)
```

Comment:
Partly for time since I'd have to override it and do a bunch of tweaking and I felt like it was maybe overkill but if we want that level of granularity I can go do it.